index.html

```diff
@@ -30,14 +30,14 @@
         </nav>
     </header>
     <section class="main--content">
-        <div class="about--container">
+        <div class="about--container main-margin">
            <h1>I’m Baraka<br>Glad to see you!</h1> 
         </div> 
-        <div class="description--container">
+        <div class="description--container main-margin">
            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
-        <p class ="supporting--text">LET’S CONNECT</p>
-        <div class="social--icons">
+        <p class ="supporting--text main-margin" main-margin>LET’S CONNECT</p>
+        <div class="social--icons main-margin">
             <ul>
                 <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
                 <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
```

Comment:
Well done in implementing the features requested by previous reviewers. Kindly add alt messages to your images, for example, alt="Twitter Icon", adding it will make your website easier to use for users with low bandwidth.